plugins/source/aws/codegen/recipes/apprunner.go

```diff
@@ -28,6 +28,62 @@ func ApprunnerResources() []*Resource {
 			Relations: []string{
 				"Operations()",
 				"CustomDomains()"},
+		}, {
+			SubService:          "auto_scaling_configuration",
+			Struct:              &types.AutoScalingConfiguration{},
+			Description:         "https://docs.aws.amazon.com/apprunner/latest/api/API_AutoScalingConfiguration.html",
+			SkipFields:          []string{"AutoScalingConfigurationArn"},
+			Multiplex:           `client.ServiceAccountRegionMultiplexer("apprunner")`,
+			PreResourceResolver: "getAutoScalingConfiguration",
+			ExtraColumns: append(
+				defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `schema.PathResolver("AutoScalingConfigurationArn")`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
+				}...),
+		},
+		{
+			SubService:  "vpc_connector",
+			Struct:      &types.VpcConnector{},
+			Description: "https://docs.aws.amazon.com/apprunner/latest/api/API_VpcConnector.html",
+			SkipFields:  []string{"VpcConnectorArn"},
+			Multiplex:   `client.ServiceAccountRegionMultiplexer("apprunner")`,
+			ExtraColumns: append(
+				defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `schema.PathResolver("VpcConnectorArn")`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
+				}...),
+		}, {
+			SubService:          "vpc_ingress_connection",
+			Struct:              &types.VpcIngressConnection{},
+			Description:         "https://docs.aws.amazon.com/apprunner/latest/api/API_VpcIngressConnection.html: account_id has been renamed to `source_account_id` to avoid conflict with the account_id column that indicates what account this was synced from.",
```

Comment:
Problem with description

Reply:
What is the issue? Normally I would put this information in the column description field but we don't use that anywhere else

Reply:
Yeah, I think it's fair to put it in the table description. Let's maybe just put the note in a separate paragraph, like:

```
`https://docs.aws.amazon.com/apprunner/latest/api/API_VpcIngressConnection.html

Notes:
 - `account_id` has been renamed to `source_account_id` to avoid conflict with the `account_id` column that indicates what account this was synced from.
```

WDYT?

Reply:
Fixed, thanks for the suggestion... Only change is that you can't have a backtick in a multiline string 😢